docs/csharp/language-reference/compiler-messages/cs9026.md

```diff
@@ -0,0 +1,31 @@
+---
+description: "Compiler Error CS9026"
+title: "Compiler Error CS9026"
+ms.date: 9/3/2022
+f1_keywords:
+  - "CS9026"
+helpviewer_keywords:
+  - "CS9026"
+ms.assetid: deeaeabd-cf52-4752-8371-82d4ad383ea3
+---
+# Compiler Error CS9026
+
+The input string cannot be converted into the equivalent UTF-8 byte representation. Unable to translate Unicode character {0} at index {1} to specified code page.
```

Comment:
We include the exact error message to help with SEO. The `{0}` and `{1}` don't help, because when a developer copies the message those are substituted with the actual values. I usually just remove those, but leave most of the message intact:

```suggestion
The input string cannot be converted into the equivalent UTF-8 byte representation. Unable to translate Unicode character to specified code page.
```

Reply:
What about a message that includes substituted values (possibly aligning with an example)?

Reply:
That's less helpful for SEO. The example values aren't likely in the developer's sample.